layouts/joomla/content/options_default.php

```diff
@@ -26,6 +29,12 @@
     <?php $fieldsnames = explode(',', $displayData->fieldsname); ?>
         <div class="form-grid">
         <?php foreach ($fieldsnames as $fieldname) : ?>
+            <?php if (array_key_exists($fieldname, $fieldsets) && property_exists($fieldsets[$fieldname], 'description') && !empty($fieldsets[$fieldname]->description)) : ?>
```

Comment:
```suggestion
            <?php if (!empty($fieldsets[$fieldname]->description)) : ?>
```
This should work the same but shorter?

Reply:
I'm seconding this.

Reply:
Thanks